frontend/packages/operator-lifecycle-manager/src/utils/useOperatorHubConfig.tsx

```diff
@@ -0,0 +1,17 @@
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { OperatorHubKind } from '../components/operator-hub';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { OperatorHubModel } from '../models';
+
+const useOperatorHubConfig = () => {
+  const [operatorHub, operatorHubLoaded, operatorHubLoadError] = useK8sWatchResource<
+    OperatorHubKind
+  >({
+    kind: referenceForModel(OperatorHubModel),
+    name: 'cluster',
+    isList: false,
+  });
+  return [operatorHub, operatorHubLoaded, operatorHubLoadError];
+};
```

Comment:
nit: this could be simplified:
```suggestion
const useOperatorHubConfig = () =>
  useK8sWatchResource<OperatorHubKind>({
    kind: referenceForModel(OperatorHubModel),
    name: 'cluster',
    isList: false,
  });
```